src/Microsoft.AspNet.HttpOverrides/project.json

```diff
@@ -0,0 +1,18 @@
+﻿{
+  "version": "1.0.0-*",
+  "description": "Various middleware for ASP.NET 5",
```

Comment:
It's not "various" anymore - we should list what's in here.

Also, I'm not sure if we should say the "ASP.NET 5" vs. just "ASP.NET". @danroth27 guidance?